ArcGISRuntimeSDKQt_QMLSamples/Analysis/DistanceMeasurementAnalysis/DistanceMeasurementAnalysis.qml

```diff
@@ -0,0 +1,227 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 1.4
+import Esri.ArcGISRuntime 100.3
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property real scaleFactor: System.displayScaleFactor
+    property bool isNavigating: false
+
+    SceneView {
+        id: sceneView
+        anchors.fill: parent
+
+        property bool isPressAndHeld: false
+
+        // Declare a Scene
+        Scene {
+            id: scene
+            // Set the basemap
+            BasemapImagery {}
+
+            // Add a Scene Layer
+            ArcGISSceneLayer {
+                url: "http://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/layers/0"
+            }
+
+            // Set the Surface
+            Surface {
+                ArcGISTiledElevationSource {
+                    url: "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"
+                }
+                ArcGISTiledElevationSource {
+                    url: "https://tiles.arcgis.com/tiles/d3voDfTFbHOCRwVR/arcgis/rest/services/MNT_IDF/ImageServer"
+                }
+            }
+
+            // set initial viewpoint
+            ViewpointCenter {
+                center: locationDistanceMeasurement.startLocation
+                targetScale: 200
+
+                Camera {
+                    location: locationDistanceMeasurement.startLocation
+                    distance: 200
+                    pitch: 45
+                    heading: 0
+                    roll: 0
+                }
+            }
+        }        
+
+        // Declare an AnalysisOverlay
+        AnalysisOverlay {
+            id: analysisOverlay
+
+
+            // Declare a Location Distance Measurement
+            LocationDistanceMeasurement {
+                id: locationDistanceMeasurement
+
+                property string unitLabel: unitSystem === Enums.UnitSystemMetric ? "m" : "ft"
+
+                // set unit system
+                unitSystem: Enums.UnitSystemMetric
+                // set the start point
+                startLocation: Point {
+                    x: -4.494677
+                    y: 48.384472
+                    z: 24.772694
+                    spatialReference: SpatialReference { wkid: 4326 }
+                }
+                // set the end point
+                endLocation: Point {
+                    x: -4.495646
+                    y: 48.384377
+                    z: 58.501115
+                    spatialReference: SpatialReference { wkid: 4326 }
+                }
+                // connect to distance change signals
+                onDirectDistanceChanged: directDistanceText.text = directDistance.value.toFixed(2) + " %1".arg(unitSystem === Enums.UnitSystemMetric ? "m" : "ft")
```

Comment:
maybe these could be bound on the UI control rather than having it this way around?